plugins/source/aws/resources/services/cloudformation/template_summaries.go

```diff
@@ -0,0 +1,56 @@
+package cloudformation
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go-v2/service/cloudformation"
+	"github.com/aws/aws-sdk-go-v2/service/cloudformation/types"
+	"github.com/cloudquery/cloudquery/plugins/source/aws/client"
+	"github.com/cloudquery/plugin-sdk/v2/schema"
+	"github.com/cloudquery/plugin-sdk/v2/transformers"
+)
+
+func templateSummaries() *schema.Table {
+	tableName := "aws_cloudformation_template_summaries"
+	return &schema.Table{
+		Name:        tableName,
+		Description: `https://docs.aws.amazon.com/AWSCloudFormation/latest/APIReference/API_GetTemplateSummary.html`,
+		Resolver:    fetchTemplateSummary,
+		Multiplex:   client.ServiceAccountRegionMultiplexer(tableName, "cloudformation"),
+		Transform: transformers.TransformWithStruct(
+			&cloudformation.GetTemplateSummaryOutput{},
+			transformers.WithSkipFields("ResultMetadata"), // This field contains metadata about the API call rather than the template itself, so remove it.
+		),
+		Columns: []schema.Column{
+			client.DefaultAccountIDColumn(false),
+			client.DefaultRegionColumn(false),
+			{
+				Name:     "stack_id",
+				Type:     schema.TypeString,
+				Resolver: schema.ParentColumnResolver("id"),
+			},
+			{
+				Name:     "metadata",
+				Type:     schema.TypeJSON,
+				Resolver: schema.PathResolver("Metadata"),
+			},
+		},
+	}
+}
+
+func fetchTemplateSummary(ctx context.Context, meta schema.ClientMeta, parent *schema.Resource, res chan<- any) error {
+	c := meta.(*client.Client)
+	svc := c.Services().Cloudformation
+
+	stack := parent.Item.(types.Stack)
+
+	summary, err := svc.GetTemplateSummary(ctx, &cloudformation.GetTemplateSummaryInput{
+		StackName: stack.StackName,
+	})
+	if err != nil {
```

Comment:
Note: not entirely sure if we should handle certain errors here or if the ones we want to ignore are already handled - e.g. stack is not in a state where getTemplateSummary succeed, because deleted etc.

Reply:
There is always a chance that the in the time between the initial request to get the stack and getting a summary here something happens (like the stack getting deleted), so we can't necessarily rely on the higher level handling it. 

That said, I think just returning the error here is fine for now. We can refine this upon further feedback if necessary, but for the time being this will make users aware of issues so they can choose to act on it (or not)